types/password-prompt.d.ts

```diff
@@ -1 +1,11 @@
-export default function(prompt: string): Promise<string>;
```

Comment:
to be honest I'm kind of surprised that this worked (and it seems to have). Can you say more about why this change was necessary? I note that `esModuleInterop` is explicitly false in this project, and `allowSyntheticDefaultImports` was not enabled.

It seems that for at least some verion(s) of `password-prompt`, consumption of a default export was fine. 

What changed?

Reply:
Almost a year old. Have no idea anymore